src/common/components/Modal/Modal.tsx

```diff
@@ -8,27 +8,39 @@ ReactModal.setAppElement('#root')
 
 export type TModal = {
   isOpen: boolean
-  handleClose?: React.MouseEventHandler<Element>
+  handleClose?: React.MouseEventHandler
   size?: 'md' | 'lg' | '3xl' | 'xl' | '4xl'
   children?: ReactNode
+  className?: string
+  overlayClassName?: string
 }
 
 const Modal = ({
   isOpen,
   handleClose,
   size,
   children,
+  className,
+  overlayClassName,
 }: TModal): JSX.Element => {
-  const modalClasses = cn({
-    'relative bg-white rounded-2xl shadow-xSmall w-full max-h-full py-8 px-12 overflow-auto mx-auto': true,
-    [`max-w-${size}`]: size,
-  })
+  const modalClasses = cn(
+    {
+      'relative bg-white rounded-2xl shadow-xSmall w-full max-h-full py-8 px-12 overflow-auto mx-auto': true,
+      [`max-w-${size}`]: size,
```

Comment:
Because of purging process, Tailwind doesn't allow to use such dynamic classes. It works fine in development, but after production build some `max-w-` may be missed.

Instead, just use className